src/debug/telemetry/Kconfig

```diff
@@ -18,3 +18,12 @@ config SOF_TELEMETRY_PERFORMANCE_MEASUREMENTS
 	  Performance records are stored in the limited number of slots in Memory Window 3,
 	  so only a certain number (PERFORMANCE_DATA_ENTRIES_COUNT) of components can be measured.
 
+config SOF_TELEMETRY_IO_PERFORMANCE_MEASUREMENTS
+	bool "enable I/O performance measurements"
+	default n
```

Comment:
"default n" is not needed for booleans -> https://docs.zephyrproject.org/latest/build/kconfig/tips.html#style-recommendations-and-shorthands